src/tiles/item-image.ts

```diff
@@ -26,13 +26,20 @@ export class ItemImage extends LitElement {
   @query('img') private baseImage!: HTMLImageElement;
 
   render() {
+    const width = () => {
+      if (this.isListTile && !this.isCompactTile) return '100px';
+      if (this.isListTile && this.isCompactTile) return '45px';
+      return '160px';
+    };
+
     return html`
       <div class=${classMap(this.itemBaseClass)}>
         <img
           class=${classMap(this.itemImageClass)}
           src="${this.imageSrc}"
           alt=""
           @load=${this.onLoad}
+          style="--imgWidth:${width()};"
```

Comment:
question: if we just get rid of this line, won't the width just work? Or am I missing something?

Reply:
i removed it now, thanks for your feedback